lib/src/core/room.dart

```diff
@@ -119,6 +121,16 @@ class Room extends DisposableChangeNotifier with EventsEmittable<RoomEvent> {
   // Agents
   final Map<String, DateTime> _transcriptionReceivedTimes = {};
 
+  // RPC Handlers
+  final Map<String, RpcRequestHandler> rpcHandlers = {};
+
+  // RPC Pending Acks
+  final Map<String, Function(String participantIdentity)> rpcPendingAcks = {};
+
+  // RPC Pending Responses
+  final Map<String, Function(String? payload, RpcError? error)>
```

Comment:
should these all be prefixed with `_` to mark them as private?

Reply:
makes sense, yeah, it should be private